doc/reference/reference_rock/vshard/vshard_ref.rst

```diff
@@ -351,6 +353,8 @@ Router public API
         * ``timeout`` – a request timeout, in seconds. In case the ``router`` cannot identify a
           shard with the bucket id, the operation will be repeated until the
           timeout is reached.
+        * other :ref:`net.box options <net_box-options>`, such as ``is_async``,
+          ``buffer``, ``on_push`` are also supprted.
```

Comment:
@Onvember please fix ALL typos of this kind.

Reply:
```suggestion
          ``buffer``, ``on_push`` are also supported.
```